### DISCUSSION TITLE
Custom Vector layer not showing

### DISCUSSION THREAD

**Discussion by dpyeates** [rnmapbox/maps#2096] [Q&A]
Hi,

I am attempting to get the OpenAIP vector tileset loaded as a Vector layer using

```
<MapboxGL.VectorSource
      id="openaipSource"
      tileUrlTemplates={[
       'https://api.tiles.openaip.net/api/data/openaip/{z}/{x}/{y}.pbf?apiKey={APIKEY}'
      ]}
/>
```

However I not getting anything displayed. What am I missing?
Log shows "{com.mapbox.common.TileStoreFile}[General]: Failed to get datasets from URL template..."

I am using v10

The OpenAIP Tile Layer documentation can be seen here:
[https://docs.openaip.net/?urls.primaryName=Tiles%20API](https://docs.openaip.net/?urls.primaryName=Tiles%20API)

**Comment by Havbaba** [comment]
Are you sure your key is an apiKey and not a "Client ID Token" ?

I can't find where to request an apiKey, I can only find the place to generate a Client ID Token..

**Reply by dpyeates** [reply]
you only need an API key if you wish to download/access the source files/data files. If you just want to access the pdf files from a client perspective you just need a client ID (at least that is my understanding) I am using the png versions of the tiles successfully, but changing to pdf, gives the above error.

**Comment by CarlosAlbertoTI** [comment]
I’ve try something similar and also didn’t work. Here it’s my example: 

In this case, I have any errors, but the layer doesn’t appear.
```
<MapboxGL.VectorSource
                        id="idTestSource"
                        tileUrlTemplates={[
                            'http://localhost:3030/cloud_test/{z}/{x}/{y}.pbf',
                        ]}
                        minZoomLevel={4}
                        maxZoomLevel={4}
 />
```
